castle/cms/browser/controlpanel/status.py

```diff
@@ -5,51 +5,64 @@
 import celery.bin.base
 import celery.bin.celery
 import celery.platforms
+import os
 
 
 class StatusView(BrowserView):
 
     def docsplit(self):
         retval = self.DocsplitSearch()
         if retval[0]:
-            status = 'OK'
-            icon = '<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: '
-            return '<span>' + icon + status + '</span>'
+            return True
         if not retval[0]:
-            status = 'ERROR: ' + retval[1]
-            icon = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Docsplit: '
-            return '<span>' + icon + status + '</span>'
+            return False
+
+    def docsplitError(self):
+        retval2 = self.DocsplitSearch()
+        if not retval2[0]:
+            return str(retval2[1])
 
     def elasticsearch(self):
         rs = redis.Redis("localhost")
+        rs2 = os.environ.get('REDIS_SERVER')
```

Comment:
You should also handle the case when this environment variable hasn't been set, and if it hasn't been set then you can reasonably assume the default value to be 'localhost'